.github/workflows/ci.yml

```
@@ -124,6 +124,14 @@ jobs:
         exit 1
       shell: powershell
     
+    - name: Build binary for tests
+      shell: bash
+      run: |
+        echo "Building binary for integration tests..."
+        go build -o reviewtask .
+        # Also copy to test directory for tests that look there
+        cp reviewtask test/reviewtask
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)